stm32-modules/heater-shaker/firmware/CMakeLists.txt

```diff
@@ -73,30 +74,22 @@ add_executable(heater-shaker
   ${HS_FW_LINTABLE_SRCS}
   ${HS_FW_NONLINTABLE_SRCS})
 
-target_link_options(heater-shaker
-  PRIVATE
-  "LINKER:-T,${SYSTEM_DIR}/STM32F303RETx_FLASH.ld"
-  "LINKER:--print-memory-usage"
-  "LINKER:--error-unresolved-symbols"
-  "LINKER:--gc-sections"
-  "LINKER:-u,_printf_float"
-  "LINKER:-u,_scanf_float")
-
 # Incurs at least a relink when you change the linker file (and a recompile of main
 # but hopefully that's quick)
 set_source_files_properties(${SYSTEM_DIR}/main.cpp
   PROPERTIES
-  OBJECT_DEPENDS "${SYSTEM_DIR}/STM32F303RETx_FLASH.ld")
+  OBJECT_DEPENDS "${COMMON_MCU_DIR}/STM32F303RETx_FLASH.ld")
 
 set_target_properties(heater-shaker
   PROPERTIES CXX_STANDARD 20
              CXX_STANDARD_REQUIRED TRUE
              C_STANDARD 11
              C_STANDARD_REQUIRED TRUE)
 
-  target_include_directories(heater-shaker
-    PUBLIC ${CMAKE_CURRENT_SOURCE_DIR}/../include
-    PRIVATE ${MOTOR_DIR})
+target_include_directories(heater-shaker
+  PUBLIC ${CMAKE_CURRENT_SOURCE_DIR}/../../include/heater-shaker
+  PUBLIC ${CMAKE_CURRENT_SOURCE_DIR}/../../include/common
```

Comment:
This should be included automatically as long as it's set as a public include dir for the common static lib